.circleci/config.yml

```diff
@@ -109,9 +109,9 @@ jobs:
           name: Test
           command: cargo test --release --offline --verbose
 
-  test_nightly:
+  test_beta:
     docker:
-      - image: rustlang/rust:nightly
+      - image: instrumentisto/rust:beta
```

Comment:
why not rustlang?

Reply:
`rustlang/rust` has only nightly and official `rust` has only stable. Beta is not that popular.

Reply:
ok thanks for providing that to us. We can merge to the IOHK's official rust docker image when available.